### PR TITLE
Add JsonProperty to ControlPlanePluginConfig constructor parameter

### PR DESCRIPTION
**Pull request by minwoox**
Modification:
- Add `JsonProperty` to `ControlPlanePluginConfig` constructor parameter

Result:
- You can now configure `ControlPlanePluginConfig` via configuration file properly.

### CODE REVIEW

**Review by jrhee17 — APPROVED** [review]
👍 👍 👍

File: xds/src/main/java/com/linecorp/centraldogma/xds/internal/ControlPlanePluginConfig.java [hunk 1]
```diff
@@ -26,7 +28,7 @@ public final class ControlPlanePluginConfig extends AbstractPluginConfig {
     /**
      * Creates a new instance.
      */
-    public ControlPlanePluginConfig(@Nullable Boolean enabled) {
+    public ControlPlanePluginConfig(@JsonProperty("enabled") @Nullable Boolean enabled) {
```
Comment (ikhoon) [inline xds/src/main/java/com/linecorp/centraldogma/xds/internal/ControlPlanePluginConfig.java:31]: Don't we need `@JsonCreator`?

Comment (minwoox) [reply xds/src/main/java/com/linecorp/centraldogma/xds/internal/ControlPlanePluginConfig.java:31]: It's not needed when there's only one constructor. Let me add it though for clarity.

**Review by trustin — APPROVED** [review]
:+1:

**Review by ikhoon — APPROVED** [review]
Thanks! 🙇‍♂️🙇‍♂️

**Pull request merged**